src/reusable-blocks/class-manage-batch.php

```diff
@@ -4,10 +4,30 @@
 
 class ManageBatch extends Manage {
 
+	/** @var Notice $notice */
+	private $notice;
+
+	public function __construct(
+		Blocks $blocks,
+		Translation $translation,
+		Notice $notice
+	) {
+		parent::__construct( $blocks, $translation );
+		$this->notice = $notice;
+	}
+
 	public function addBlocks( \WPML_TM_Translation_Batch $batch ) {
 		$blocks = $this->getBlocksFromPostElements( \collect( $batch->get_elements() ) );
 
-		$this->getBlockElementsToAdd( $blocks )->each(
+		$blocks_to_add = $this->getBlockElementsToAdd( $blocks );
+
+		if ( $blocks_to_add->count() < 1 ) {
```

Comment:
@strategio Why `< 1`? Wouldn't `if ( $blocks_to_add->isEmpty() ) {` be better?

Reply:
Nice one @brucepearson :)